src/main/java/frc/robot/RobotContainer.java

```diff
@@ -85,6 +95,26 @@ private void configureBindings() {
     m_joystick.button(2).onTrue(m_intaketoouttakecommand);
     m_joystick.button(3).onTrue(m_stowcommand);
     */
+    
+// Since we are using a holonomic drivetrain, the rotation component of this pose
+// represents the goal holonomic rotation
+Pose2d targetPose = new Pose2d(10, 5, Rotation2d.fromDegrees(180));
+
+// Create the constraints to use while pathfinding
+PathConstraints constraints = new PathConstraints(
+        3.0, 4.0,
+        Units.degreesToRadians(540), Units.degreesToRadians(720));
+
+// Since AutoBuilder is configured, we can use it to build pathfinding commands
+Command pathfindingCommand = AutoBuilder.pathfindToPose(
+        targetPose,
+        constraints,
+        0.0, // Goal end velocity in meters/sec
+        0.0 // Rotation delay distance in meters. This is how far the robot should travel before attempting to rotate.
+        
+        
+);
```

Comment:
All of this code for pathfinding is in the configureBindings() method when it should probably be at the top of the class with the other fields.